k8s/stork/backuplocation.go

```diff
@@ -68,6 +72,12 @@ func (c *Client) ListBackupLocations(namespace string, filterOptions metav1.List
 			return nil, err
 		}
 	}
+	for i := range backupLocations.Items {
+		err = backupLocations.Items[i].UpdateFromClusterSecret(c.kube)
```

Comment:
nit : As there is a for loop in the line#69, maybe this can be moved there itself.

Reply:
fixed.